src/catalog/src/durable/persist.rs

```diff
@@ -220,6 +220,21 @@ impl FenceableToken {
         }
     }
 
+    /// Returns the durable epoch.
+    fn durable_epoch(&self) -> Option<Epoch> {
+        match self {
+            FenceableToken::Initializing {
+                durable_token,
+                legacy_durable_epoch,
+                ..
+            } => durable_token
+                .clone()
+                .map(|token| token.epoch)
+                .or(legacy_durable_epoch.clone()),
```

Comment:
This branch is technically never taken in the code, meaning this change is technically a no-op. Still, better safe than sorry.